### PR TITLE
connections may updates

### PR DESCRIPTION
**Pull request by meganelepalud-akeneo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "dependencies": {
     "bootstrap": "3.4.1",
     "docsearch.js": "2.4.1",
-    "jquery": "3.4.0",
+    "jquery": "^3.5.1",
```
Comment (NolwennP) [inline package.json:6]: You upgraded the dependencies on purpose?

File: .gitignore [hunk 2]
```diff
@@ -5,3 +5,4 @@
 .DS_Store
 /tmp/
 .env
+package-lock.json
```
Comment (NolwennP) [inline .gitignore:8]: We will commit this file finally.

File: content/updates/2020-05/1-new-connection-monitoring-page.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- and connection cards with the number of business errors during the past week with a link to the errors list of the related connection.
+
+![Error monitoring on connection dashboard](../img/error-monitoring-connection-dashboard.png)
+
+::: warning
+The feature is only available for Sources connections. 
+:::
+
+::: more
+[What is a connection?](../articles/what-is-a-connection.html) [How to monitor your third-party connections?](../articles/connection-dashboard.html) [How to manage your connections?](../articles/manage-your-connections.html)
```
Comment (NolwennP) [inline content/updates/2020-05/1-new-connection-monitoring-page.md:28]:
```suggestion
[What is a connection?](../articles/what-is-a-connection.html)  
[How to monitor your third-party connections?](../articles/connection-dashboard.html)  
[How to manage your connections?](../articles/manage-your-connections.html) 
```

File: content/updates/2020-05/1-new-connection-monitoring-page.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# New: monitor products errors on your connections
+
+## A new page to monitor your connection errors
+::: meta-data type="New" features="Connections" available="mid-May" link-to-doc="../articles/manage-your-connections.html#monitor-a-connection"
```
Comment (NolwennP) [inline content/updates/2020-05/1-new-connection-monitoring-page.md:4]: Is this really the link to the documentation of this new feature?

File: content/updates/2020-05/1-new-connection-monitoring-page.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+:::
+
+This feature should help you to reduce the time needed to solve issues. The goal of displaying product business errors is to allow you to solve them before calling for IT support or your integrator support.
+
+In other words, why a product hasn't been created or updated by your source connection?
+
+![Connection monitoring page](../img/new-connection-monitoring-page.png)
+
+## New metrics on the connection dashboard
+::: meta-data type="New" features="Connections" available="late-May" link-to-doc="../articles/manage-your-connections.html#monitor-a-connection"
```
Comment (NolwennP) [inline content/updates/2020-05/1-new-connection-monitoring-page.md:14]: Is this really the link to the documentation of this new feature?

File: content/updates/2020-05/1-new-connection-monitoring-page.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+:::
+
+The connection dashboard now displays: 
+- a new graph with the number of product errors (business and technical) for the past week,
+- and connection cards with the number of business errors during the past week with a link to the errors list of the related connection.
```
Comment (NolwennP) [inline content/updates/2020-05/1-new-connection-monitoring-page.md:19]: What are "connections cards"?

**Comment by NolwennP** [comment]
Don't forget to put Laure and Camille as reviewers whenever you're done with the conten. 😉

File: package.json (hunk 1 above)
Comment (meganelepalud-akeneo) [reply package.json:6]: No, I don't know why I committed these updates... I did a `npm audit fix` after the `npm install`, it's probably this that updated dependencies. I will remove this file from my commit

File: content/updates/2020-05/1-new-connection-monitoring-page.md (hunk 6 above)
Comment (meganelepalud-akeneo) [reply content/updates/2020-05/1-new-connection-monitoring-page.md:19]: We added "cards" on the dashboard. One for each source connection that displays connection icon, name, # of business errors for the last 7 days. I added the screenshot to illustrate.

File: content/updates/2020-05/1-new-connection-monitoring-page.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
+# New: monitor products errors on your connections
```
Comment (LaureBro) [inline content/updates/2020-05/1-new-connection-monitoring-page.md:1]: Title must be for both new features
```suggestion
# Connections new features
```

File: content/updates/2020-05/1-new-connection-monitoring-page.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+## A new page to monitor your connection errors
+::: meta-data type="New" features="Connections" available="mid-May" link-to-doc="../articles/connection-dashboard.html#error-monitoring"
```
Comment (LaureBro) [inline content/updates/2020-05/1-new-connection-monitoring-page.md:4]:
```suggestion
::: meta-data type="New" features="Connections, Productivity" available="mid-May" link-to-doc="../articles/connection-dashboard.html#error-monitoring"
```

File: content/updates/2020-05/1-new-connection-monitoring-page.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
+# New: monitor products errors on your connections
+
+## A new page to monitor your connection errors
```
Comment (LaureBro) [inline content/updates/2020-05/1-new-connection-monitoring-page.md:3]:
```suggestion
## Easily monitor errors on your connections 
```

File: content/updates/2020-05/1-new-connection-monitoring-page.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+::: meta-data type="New" features="Connections" available="mid-May" link-to-doc="../articles/connection-dashboard.html#error-monitoring"
+:::
+
+This feature should help you to reduce the time needed to solve issues. The goal of displaying product business errors is to allow you to solve them before calling for IT support or your integrator support.
```
Comment (LaureBro) [inline content/updates/2020-05/1-new-connection-monitoring-page.md:7]:
```suggestion
The new `Monitoring` page available under `Connections` lists the last errors for your Connections. From this page, you can now easily track any error on products creation, update, or deletion for each of your connections and fix it. 
```

File: content/updates/2020-05/1-new-connection-monitoring-page.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+
+In other words, why a product hasn't been created or updated by your source connection?
```
Comment (LaureBro) [inline content/updates/2020-05/1-new-connection-monitoring-page.md:9]: I'd remove this sentence

File: content/updates/2020-05/1-new-connection-monitoring-page.md [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+
+![Connection monitoring page](../img/new-connection-monitoring-page.png)
+
+## New metrics on the connection dashboard
+::: meta-data type="New" features="Connections" available="late-May" link-to-doc="../articles/connection-dashboard.html#error-monitoring"
```
Comment (LaureBro) [inline content/updates/2020-05/1-new-connection-monitoring-page.md:14]:
```suggestion
::: meta-data type="New" features="Connections, Productivity" available="late-May" link-to-doc="../articles/connection-dashboard.html#error-monitoring"
```

File: content/updates/2020-05/1-new-connection-monitoring-page.md [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+
+The connection dashboard now displays: 
+- a new graph with the number of product errors (business and technical) for the past week,
+- and connection cards with the number of business errors during the past week with a link to the errors list of the related connection.
+
+![Error monitoring on connection dashboard](../img/error-monitoring-connection-dashboard.png)
+
+::: warning
+The feature is only available for Sources connections. 
+:::
```
Comment (LaureBro) [inline content/updates/2020-05/1-new-connection-monitoring-page.md:25]: remove the warning part - no need to add it in the update article. But it needs to be in the help center article

File: content/updates/2020-05/1-new-connection-monitoring-page.md [hunk 14]
```diff
@@ -0,0 +1,31 @@
...
+::: meta-data type="New" features="Connections" available="mid-May" link-to-doc="../articles/connection-dashboard.html#error-monitoring"
+:::
+
+This feature should help you to reduce the time needed to solve issues. The goal of displaying product business errors is to allow you to solve them before calling for IT support or your integrator support.
+
+In other words, why a product hasn't been created or updated by your source connection?
+
+![Connection monitoring page](../img/new-connection-monitoring-page.png)
+
+## New metrics on the connection dashboard
```
Comment (LaureBro) [inline content/updates/2020-05/1-new-connection-monitoring-page.md:13]: Connection with a C-cap ?

File: content/updates/2020-05/1-new-connection-monitoring-page.md [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+::: meta-data type="New" features="Connections" available="late-May" link-to-doc="../articles/connection-dashboard.html#error-monitoring"
+:::
+
+The connection dashboard now displays:
```
Comment (LaureBro) [inline content/updates/2020-05/1-new-connection-monitoring-page.md:17]:
```suggestion
To ease the error monitoring and enable you to see at a glance how your Source Connections are performing, the connection dashboard now displays: 
```

File: content/updates/2020-05/1-new-connection-monitoring-page.md [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+In other words, why a product hasn't been created or updated by your source connection?
+
+![Connection monitoring page](../img/new-connection-monitoring-page.png)
+
+## New metrics on the connection dashboard
+::: meta-data type="New" features="Connections" available="late-May" link-to-doc="../articles/connection-dashboard.html#error-monitoring"
+:::
+
+The connection dashboard now displays: 
+- a new graph with the number of product errors (business and technical) for the past week,
```
Comment (LaureBro) [inline content/updates/2020-05/1-new-connection-monitoring-page.md:18]:
```suggestion
- a graph with the number of product errors (business and technical) for the past week,
```

File: content/updates/2020-05/1-new-connection-monitoring-page.md [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+- and connection cards with the number of business errors during the past week with a link to the errors list of the related connection.
```
Comment (LaureBro) [inline content/updates/2020-05/1-new-connection-monitoring-page.md:19]:
```suggestion
- and connection cards showing the number of business errors for the past week. 
```

**Pull request merged**